crates/pop-parachains/templates/base/runtime-cargo.templ

```diff
@@ -1,74 +1,75 @@
 [package]
```

Comment:
runtime-cargo.templ and runtime.templ are not being used. My suggestion would be remove them to don't mantain dead code